### PR TITLE
add methods to retrieve and set baseline

### PR DESCRIPTION
**Pull request by coding-guy**
just simple get and set the baseline value

### CODE REVIEW

**Comment by bfaliszek** [comment]
@coding-guy Hi, do you have an example of code using read and write baseline?

File: Adafruit_CCS811.cpp [hunk 1]
```diff
@@ -273,3 +273,41 @@ void Adafruit_CCS811::write(uint8_t reg, uint8_t *buf, uint8_t num)
 	Wire.write((uint8_t *)buf, num);
 	Wire.endTransmission();
 }
+
+//Returns the baseline value
+//Used for telling sensor what 'clean' air is
+//You must put the sensor in clean air and record this value
+uint16_t Adafruit_CCS811::getBaseline( void )
```
Comment (siddacious) [inline Adafruit_CCS811.cpp]: This function needs to be documented in the Doxygen style. See this guide for help:
https://learn.adafruit.com/the-well-automated-arduino-library/doxygen
I would highly recommend running Doxygen locally before pushing more commits

File: Adafruit_CCS811.cpp [hunk 2]
```diff
@@ -273,3 +273,41 @@ void Adafruit_CCS811::write(uint8_t reg, uint8_t *buf, uint8_t num)
...
+	{
+		return 0;
+	}
+	else
+	{
+		return (baseline);
+	}
+}
+
+bool Adafruit_CCS811::setBaseline( uint16_t input )
```
Comment (siddacious) [inline Adafruit_CCS811.cpp]: This function needs Doxygen compatible documentation as well.

**Comment by tfilo** [comment]
Hi is there any progress in this implementation? I would like to save and restore baseline for CSS811.

**Comment by ladyada** [comment]
CI-passing code was recently merged, check it out!

**Pull request closed without merging**